### ISSUE TITLE
push and pop additional beetle properties

### ISSUE THREAD

**Issue by ericrosenbaum** [ericrosenbaum/BeetleBlocks#166]
arguably, push and pop could also include the other beetle state variables:

* Extrusion diameter
* Color

What do you think @gitduks?

**Comment by gitduks** [comment]
I agree regarding the color. If we include the 'diameter', we should have an indicator at the bottom right of the screen. I actually thought we could extend this even further and include the 'drawing state' (similar to pen down or up)
